src/python/grpcio_tests/tests/tests.json

```diff
@@ -25,7 +25,8 @@
   "_implementations_test.CallCredentialsTest",
   "_implementations_test.ChannelCredentialsTest",
   "_insecure_interop_test.InsecureInteropTest",
-  "_invalid_metadata_test.InvalidMetadataTest"
+  "_invalid_metadata_test.InvalidMetadataTest",
```

Comment:
Er, hm. That makes this file invalid JSON. Might be worth just patching that in another PR as soon as possible.